automated/utils/test-runner.py

```diff
@@ -15,6 +16,10 @@
 import time
 from uuid import uuid4
 from distutils.spawn import find_executable
+from squad_client.core.api import SquadApi
```

Comment:
I think this should not be mandatory. Please add try-catch block around squad_client imports.

Reply:
Done